From fbd334d7bda64c7f1cec8cf6667cc6ac8f902952 Mon Sep 17 00:00:00 2001
From: "Kevin F. Haggerty" <haggertk@lineageos.org>
Date: Mon, 25 Dec 2017 15:51:39 -0700
Subject: [PATCH 06/17] klte-common: Update
 RIL_REQUEST_QUERY_AVAILABLE_NETWORKS response prop

* mQANElements never made tremendous sense, the response strings
  are not part of an object, just a text string making up a parcel.
* ro.ril.telephony.mqanelements -> ro.ril.telephony.qan_resp_strings
  for consistency with some of the other new RIL props.

Change-Id: I6f941f5177852454a4d4b9494077329814d884f1
---
 system.prop | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/system.prop b/system.prop
index 3caa653..de5476f 100644
--- a/system.prop
+++ b/system.prop
@@ -50,7 +50,7 @@ persist.data.qmi.adb_logmask=0
 persist.radio.add_power_save=1
 rild.libpath=/system/vendor/lib/libsec-ril.so
 ro.telephony.mms_data_profile=5
-ro.ril.telephony.mqanelements=6
+ro.ril.telephony.qan_resp_strings=6
 
 # Ril sends only one RIL_UNSOL_CALL_RING, so set call_ring.multiple to false
 ro.telephony.call_ring.multiple=0
-- 
2.7.4

